pandas/io/json/normalize.py

```diff
@@ -25,9 +26,10 @@ def _convert_to_line_delimits(s):
     return convert_json_to_lines(s)
 
 
-def nested_to_record(ds, prefix="", sep=".", level=0):
+def nested_to_record(ds, prefix: str = "", sep: str = ".", level: int = 0,
+                     max_level: int = None):
```

Comment:
type should be ``max_level: Optional[int] = None``

all of these need the Optional